puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pipelines/katello.groovy

```diff
@@ -33,8 +36,8 @@ pipeline {
             steps {
                 script {
                     runCicoJobsInParallel([
-                        ['name': 'Install test', 'job': "foreman-katello-${katello_version}-test"],
-                        ['name': 'Upgrade test', 'job': "foreman-katello-upgrade-${katello_version}-test"]
+                        ['name': 'centos7 install test', 'job': "foreman-katello-${katello_version}-test"],
+                        ['name': 'centos7 upgrade test', 'job': "foreman-katello-upgrade-${katello_version}-test"]
```

Comment:
Have you considered renaming the jobs to include the version in the name, like we do in Foreman? Or is that a future refactor?

Reply:
Hah, I was just writing a comment. There is a larger set of changes I started making to bring cohesion to our job naming conventions and reduce overhead of declaring pipeline jobs. So I'm trying to do as best I can smaller targeted changes to build up to that point.

Reply:
I suspected as much.